### DISCUSSION TITLE
How to input my own numbers in Amplitude?

### DISCUSSION THREAD

**Discussion by Megan8821** [magenta/ddsp#468] [Q&A]
Hi, I can't understand what does ```Amplitude [batch, n_frames, 1]``` mean in this model. I mean I want to input my own array from the wave file, but I don't know how to do it. Can someone give me an example? Thanks!

**Comment by jesseengel** [comment]
Hi Megan,

That is the amplitude of the harmonic synthesizer, so it's literally a number to multiply the sine-waves by at each moment in time. So if it's 0, it should be silent, and if it's 1.0 it should be right about to clip. Time is chopped up into frames that are a lower rate than the audio, so each frame needs an amplitude value (n_frames). The model can also work on several files at once in a "batch", which is (n_batch), but if you just have a single file, that would just be 1. 

So for instance if you have a 4-second long piece of audio you're generating, and the frame rate is like 250Hz you'd need an array of shape [1, 1000, 1] to feed for amplitudes (4 * 250 = 1000)
